### DISCUSSION TITLE
A function that stretches the edges of borderless windows.

### DISCUSSION THREAD

**Discussion by feb-cloud** [flet-dev/flet#4056] [Ideas]
**Conditions:**
_Old version_
`page.window_title_bar_hidden = True`
`page.window_frameless = True`
_New version_
`page.window.title_bar_hidden = True`
`page.window.frameless = True`

**Expectations/ideas:**
When hiding the title bar and creating a borderless window, I hope that the window border can still achieve the original system's mouse click and drag the border edge to modify the window size. This design will make our window more flexible. I hope to adopt this idea.
Or, can you teach me how I should modify the code in the current version to achieve this effect?

Looking forward to your reply, thank you.

**Comment by ndonkoHenri** [comment]
Try the [`WindowDragArea`](https://flet.dev/docs/controls/windowdragarea).
